src/nvim/tui/tui.c

```diff
@@ -1658,6 +1659,37 @@ static void out(void *ctx, const char *str, size_t len)
   data->bufpos += len;
 }
 
+/// Called by unibi_format() for padding.
```

Comment:
hard to find the docs for `unibi_format` and its `pad` param. Here are the full docs from https://github.com/neovim/unibilium/blob/6688d83c91501d81fec9a1a0fbf67993d2d338f0/doc/unibi_format.pod#L32


```
I<pad> is used when the format string contains C<< $<...> >> padding
instructions. In the calls to I<pad> the first argument is always I<ctx2>, the
second argument is the delay in tenths of milliseconds, the third argument is a
boolean flag indicating whether C<*> (proportional delay) was specified in the
format string, and the fourth argument is a boolean flag indicating whether
C</> (forced padding) was specified in the format string. Thus a format string
of C<< $<5/> >> would translate into C<pad(ctx2, 50, 0, 1)>. You may pass a
null pointer for I<pad>; in that case padding instructions are silently
skipped.
```

can we summarize the relevant part here (very concisely)? Something like:

> The `pad` parameter of `unibi_format` handles padding instructions. If it is NULL then padding instructions are silently skipped.

Reply:
I guess it is better to add doxygen comments for the parameters instead.